### ISSUE TITLE
-q/--quiet flag does not work

### ISSUE THREAD

**Issue by Hendrikto** [aurapm/aura#470]
- `aura-bin`: 1.4.0
- `pacman`: 5.0.2
- `libalpm`: 10.0.2

The `-q/--quiet` option has no effects whatsoever.

All of the following commands produce exactly the same output, but they should not.

```
aura -Ai google-chrome
aura -q -Ai google-chrome
aura --quiet -Ai google-chrome
aura -Aiq google-chrome
aura -Ai -q google-chrome
aura -Ai --quiet google-chrome
aura -Ai google-chrome -q
aura -Ai google-chrome --quiet
```

Expected result: When using the `-q/--quiet` option the output coloring should be disabled.
Observed result: The `-q/--quiet` option has no effect.

**Comment by fosskers** [comment]
Odd, considering that's what's advertised in the man page. I'm looking into it.

**Comment by fosskers** [comment]
It looks like `-q` isn't considered at all for `-i`. For `-s`, it is:
```
colin@yumi ~/c/h/aura> aura -As google-chrome
aur/google-chrome 62.0.3202.94-1 (1611 | 26.62)
    The popular and trusted web browser by Google (Stable Channel)
aur/google-chrome-dev 64.0.3278.0-1 (613 | 3.36)
    The popular and trusted web browser by Google (Dev Channel)
aur/google-chrome-beta 63.0.3239.70-1 (333 | 2.20)
    The popular and trusted web browser by Google (Beta Channel)

colin@yumi ~/c/h/aura> aura -Asq google-chrome
google-chrome
google-chrome-dev
google-chrome-beta
```

**Comment by fosskers** [comment]
The colour comment is a reflection of pacman's underlying `-s` behaviour. You'd see the difference between `-Ss` and `-Ssq` too.

**Comment by fosskers** [comment]
Is the coloured output from `-Ai` breaking something in your workflow?

**Comment by Hendrikto** [comment]
Yes, I want to parse the output with regular expressions and the coloring is interfering with that.

**Comment by fosskers** [comment]
Understood, I'll see what I can do.

**Comment by fosskers** [comment]
This is fixed on `master` and will be included with the upcoming Aura 2.0 release. `--color never` can be used to turn off all output text colouring.
